actors/abi/sector.go

```diff
@@ -32,25 +32,26 @@ type SectorWeight = big.Int
 type StoragePower = big.Int
 
 func NewStoragePower(n int64) StoragePower {
-	return StoragePower(big.NewInt(n))
+	return big.NewInt(n)
 }
 
 // This ordering, defines mappings to UInt in a way which MUST never change.
 type RegisteredProof int64
 
+//noinspection GoUnusedConst
 const (
-	RegisteredProof_WinStackedDRG32GiBSeal = RegisteredProof(1)
-	RegisteredProof_WinStackedDRG32GiBPoSt = RegisteredProof(2)
-	RegisteredProof_StackedDRG32GiBSeal    = RegisteredProof(3)
-	RegisteredProof_StackedDRG32GiBPoSt    = RegisteredProof(4)
-	RegisteredProof_StackedDRG1KiBSeal     = RegisteredProof(5)
-	RegisteredProof_StackedDRG1KiBPoSt     = RegisteredProof(6)
-	RegisteredProof_StackedDRG16MiBSeal    = RegisteredProof(7)
-	RegisteredProof_StackedDRG16MiBPoSt    = RegisteredProof(8)
-	RegisteredProof_StackedDRG256MiBSeal   = RegisteredProof(9)
-	RegisteredProof_StackedDRG256MiBPoSt   = RegisteredProof(10)
-	RegisteredProof_StackedDRG1GiBSeal     = RegisteredProof(11)
-	RegisteredProof_StackedDRG1GiBPoSt     = RegisteredProof(12)
+	RegisteredProofWinStackedDRG32GiBSeal = RegisteredProof(1)
```

Comment:
winstacked is not a valid algorithm anymore, why is this still here?

Reply:
Because no-one responsible who knows that updated the spec.